metering/metering-upgrading-metering.adoc

```diff
@@ -70,11 +70,11 @@ You can verify the metering upgrade by performing any of the following checks:
 $ oc get csv | grep metering
 ----
 +
-.Example output for metering upgrade from 4.5 to 4.6
+.Example output for metering upgrade from 4.6 to 4.7
 [source,terminal]
 ----
-NAME                                        DISPLAY                  VERSION                 REPLACES                               PHASE
-metering-operator.4.6.0-202007012112.p0     Metering                 4.6.0-202007012112.p0   metering-operator.4.5.0-202005252114   Succeeded
+NAME                                        DISPLAY                  VERSION                 REPLACES                                 PHASE
+metering-operator.4.7.0-202007012112.p0     Metering                 4.7.0-202007012112.p0   metering-operator.4.6.0-202007012112.p0  Succeeded
```

Comment:
I'll double check these with the Metering team. I might go in a manipulate the creation dates here for the upgraded operator because it's showing as mid-2020.

Reply:
@ahardin-rh These updates look good. Once this PR is merged, I'll go in and "update" the dates in these examples to be more current.

Reply:
Thank you!